optuna/study.py

```diff
@@ -322,6 +322,19 @@ def _should_stop() -> bool:
     def set_user_attr(self, key: str, value: Any) -> None:
         """Set a user attribute to the study.
 
+        Example:
```

Comment:
How about marking this example as a doctest by adding `.. testcode::`?

Reply:
Thanks for swift review!
I agree with you!